app/Console/Commands/PollTwitter.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\User;
+use Illuminate\Console\Command;
+use Log;
+
+class PollTwitter extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'suggestive:poll-twitter';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Polls Twitter for any tweets with the configured hashtag.';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $users = User::where('role', 'owner')
+                     ->whereNotNull('hashtag')
+                     ->get([ 'id', 'name', 'hashtag', 'last_tweet_id', ]);
+
+        if ($users) {
+            $twitter = app('TwitterClient');
+
+            Log::info('Found owners to poll Twitter for', [ 'count' => $users->count(), ]);
+
+            $users->each(function ($user) use ($twitter) {
```

Comment:
What's the theory behind `Job` vs scheduled `Command`? What would the trigger be to the queue the job to execute? Or are you just suggesting that I queue the request and response part of the command so that it doesn't need to handle that part of it?